common/djangoapps/student/management/commands/bulk_unenroll.py

```diff
@@ -0,0 +1,62 @@
+import logging
+import csv
+
+from django.core.exceptions import ObjectDoesNotExist
+from django.core.management.base import BaseCommand, CommandError
+from django.db.models import Q
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+
+from student.models import CourseEnrollment, User
+
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+class Command(BaseCommand):
+
+    help = """"
+    Un-enroll bulk users from the courses.
+    It expect that the data will be provided in a csv file format with
+    first row being the header and columns will be as follows:
+    user_id, username, email, course_id, is_verified, verification_date
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument('-p', '--csv_path',
+                            metavar='csv_path',
+                            dest='csv_path',
+                            required=True,
+                            help='Path to CSV file.')
+
+    def handle(self, *args, **options):
+        csv_path = options['csv_path']
+        with open(csv_path, 'rb') as csvfile:
+            reader = csv.DictReader(csvfile)
+            for row in reader:
+                username = row['username']
```

Comment:
Does it make sense to check how many columns are in the row, so if someone forgets to add a line-break, it will report that a line was too long?

Reply:
Are these csv files manually or automatically generated?

If automatic, maybe this is overkill.  If manual, could you at least run a manual test with a csv with two users on the same line (e.g. with no `\n` in between), and report back what it does?  Does it error out?  Does it silently skip the second one?  You can have the second one be an error case to better know.  Or, you could just write a unit test.

Note: I'm not going to be the user of this script, and I'm not exactly clear on why I was asked to review this PR, so I think it makes sense for someone else to decide on what is good enough and what is overkill.

Reply:
I think these CSVs are auto generated.